zomes/profile/src/lib.rs

```diff
@@ -65,10 +55,40 @@ fn validate(op: Op) -> ExternResult<ValidateCallbackResult> {
                     ..
                 },
             ..
-        } => validation::__validate_entry(new_entry, &header.author),
+        } => {
+            if is_editable() {
+                validation::__validate_entry(new_entry, &header.author)
+            } else {
+                Ok(ValidateCallbackResult::Invalid(
+                    "Invalid try to Delete Entry".to_string(),
+                ))
+            }
+        }
         Op::RegisterDeleteLink { .. } => Ok(ValidateCallbackResult::Invalid(
-            "Cannot update Link".to_string(),
+            "Invalid try to update Link".to_string(),
         )),
         _ => Ok(ValidateCallbackResult::Valid),
     }
 }
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes, Clone)]
+struct Props {
+    is_editable_profile: bool,
+}
+
+/// Checking properties for `is_editable_profile` flag
+pub fn is_editable() -> bool {
+    if let Ok(info) = dna_info() {
+        return is_editable_sb(&info.properties);
+    }
+    true
+}
+
+/// Deserialize properties into the Props expected by this zome
+pub fn is_editable_sb(encoded_props: &SerializedBytes) -> bool {
+    let maybe_props = Props::try_from(encoded_props.to_owned());
+    if let Ok(props) = maybe_props {
+        return props.is_editable_profile;
+    }
+    true
+}
```

Comment:
not feedback for this PR, just noticing this is a horrible amount of work to have to do to get a boolean dna property 😅